Gemfile.lock

```diff
@@ -522,4 +518,4 @@ DEPENDENCIES
   wirble
 
 BUNDLED WITH
-   1.15.4
```

Comment:
Are we updating this? I know it's automatic but just checking it.

Reply:
Sigh... This always hits me. I'll update it, it shouldn't be like this 👍